src/stories/Library/banner/banner.scss

```
@@ -0,0 +1,65 @@
+@mixin banner-height {
+  height: 500px;
+  @include media-query__medium {
+    height: 810px;
+  }
+}
+
+@mixin banner-text-margin {
+  margin-bottom: $s-md;
+  @include media-query__medium {
+    margin-bottom: $s-xl;
+  }
+}
+
+.banner {
+  @include layout-container($layout__max-width--large, 0);
+  @include banner-height;
+  display: grid;
+  grid-template-rows: 1fr auto 1fr;
+  text-decoration: none;
+}
+
+.banner-content {
+  padding: $s-xl;
+  max-width: 375px;
+  background-color: white;
+  grid-row: 2;
+  grid-column: 1;
+
```

In small mobile or less, there is no space on right side. I think you could do something likes, up to you wheter to use % or something else. Remember to change it on larger sizes if necessary. 

```suggestion
.banner-content {
  padding: $s-xl;
  max-width: 375px;
  background-color: white;
  grid-row: 2;
  grid-column: 1;
  width: calc(100% - 10%);
```

---

Thanks 🙏